packages/nouns-webapp/src/components/AuctionActivity/AuctionActivity.module.css

```diff
@@ -48,11 +48,10 @@
   overflow: hidden;
 }
 
-
 @media (max-width: 992px) {
   .floatingPaper {
-    border-bottom-left-radius: 42px;
-    border-bottom-right-radius: 42px;
+    border-top-left-radius: 42px;
```

Comment:
This was correct, when entering mobile mode, the goal is to round the whole box.